.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -11,35 +11,35 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-name: 🐛Bug
+name: Bug Report
 description: Have you got a bug? Please report it here!
+title: "[Bug]: "
+labels: ["bug", "triage"]
 projects: [ "SAFEHR-data/13" ]
 body:
   - type: markdown
     attributes:
       value: |
         A clear and concise description of what the bug is. Please use a short, concise title for the bug and elaborate here
   - type: textarea
-    id: reproducing_bug
+    id: actual_behaviour
     attributes:
-      label: What did you do?
-      description: What can someone see/do when this task is done.
-      placeholder: "- [ ] Please provide a minimal working example (code, system info, terminal logs). If possible: https://stackoverflow.com/help/minimal-reproducible-example"
+      label: What did you get?
```

I think this might be a bit more clear
```suggestion
      label: What happened?
```